stream-chat-android-ui-common/src/main/java/io/getstream/chat/android/common/state/MessageListAction.kt

```diff
@@ -1,4 +1,4 @@
-package io.getstream.chat.android.compose.state.messages.list
+package io.getstream.chat.android.common.state
```

Comment:
- These new `io.getstream.chat.android.common` files are under the `java` source folder instead of `kotlin`
- For this specific file, maybe the `MessageAction` name would be more appropriate, to match the name of the class